dashboard/app/views/levels/_form.html.haml

```diff
@@ -1,102 +0,0 @@
-- content_for(:head) do
-  %script{src: webpack_asset_path('js/levelbuilder.js')}
-
-%pre#validation-error.validation-error{style: 'background-color: yellow; display: none'}
```

Comment:
looks like you lost this line below. Not sure if intentional

Reply:
Sorry, yes, this was intentional. I did this in a separate commit with a descriptive name, but forgot to mention it in the PR description.